lib/features/pokemon_overview/widget/pokemon_grid_view.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:pokedex_start/api/model/pokemon.dart';
+import 'package:pokedex_start/features/pokemon_overview/widget/pokemon_card.dart';
+import 'package:pokedex_start/utilities/constant.dart';
+
+import 'package:flutter/material.dart';
+
+class PokemonGridView extends StatelessWidget {
+  const PokemonGridView({
```

Comment:
no need for this class anymore, call this directly inside your pokemon_overview_page 🙂